app/views/templates/base.php

```diff
@@ -65,16 +78,13 @@
 <html lang="en" dir="ltr">
   <head>
     <title><?php if ($show_title == true) {
-    echo $page_title . ' | ';
+    print $page_title . ' | ';
 } ?><?=$title_productname?></title>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="/style/transvision.css?<?php echo VERSION; ?>" type="text/css" media="all" />
-    <?php
-    foreach ($css_include as $css_file) {
-        echo "<link rel=\"stylesheet\" href=\"/style/{$css_file}?" . VERSION . "\" type=\"text/css\" media=\"all\" />\n";
-    }
-    ?>
+<?php foreach ($css_include as $css_file):?>
```

Comment:
I am experimenting with Twig these days (it is Jinja for PHP, originally by the author of Jinja), eventually we could switch to it but it will require rewriting all views to use that.